backend/src/main/kotlin/vegait/rs/osipodgorica/service/CategoryService.kt

```diff
@@ -19,11 +25,42 @@ class CategoryService(val categoryRepo: CategoryRepository, val uploadService: I
         return categoryRepo.save(category)
     }
 
+    fun update(request: UpdateCategoryRequest, id: Long) =
+        runCatching {
+            categoryRepo.findById(id).get()
+        }.mapCatching { oldCategory ->
+            Category(
```

Comment:
mislim da bi bilo lakše i kraće da izmapiramo samo stvari koje se mogu promijeniti, tj. name i relativeUrl. Možeš pogledati primjere u drugim servisima

Reply:
Pa samo njih i mapiram i update icokice je opcion, jedino to